static/app/utils/profiling/frame.tsx

```diff
@@ -66,12 +66,12 @@ export class Frame {
 
     // If the frame is a web frame and there is no name associated to it, then it was likely invoked as an iife or anonymous callback as
     // most modern browser engines properly show anonymous functions when they are assigned to references (e.g. `let foo = function() {};`)
-    if (type === 'web' || type === 'node') {
+    if (type === 'javascript' || type === 'node') {
       if (this.name === '(garbage collector)' || this.name === '(root)') {
         this.is_application = false;
       }
 
-      if (!this.name || this.name.startsWith('unknown ')) {
```

Comment:
Are we tightening the condition to match on `unknown` exactly here? Does this break any existing profiles?

Reply:
No, afaik the only case where this would break is if you actually have a function called unknown. I think that is highly unlikely and ok to dismiss (we also don't have a way to detect that). The proper thing here would be to either send "" for unknown functions or handle this in symbolicator/ingest (not sure where is best)

Reply:
For all other platforms, we use `""` for unknown frames, so it may be better if nodejs/javascript aligns with this as well.

Reply:
Oh, I thought that unknown is used everywhere 🤔 I will check where that value comes from and report to the team